symbols/var_meter_horz.ts

```diff
@@ -0,0 +1,32 @@
+import { defineSymbol } from "drawing/defineSymbol"
+import svgJson from "assets/generated/var_meter.json"
+
+const { paths, texts, bounds, refblocks, circles } = svgJson
+
+export default defineSymbol({
+  primitives: [
+    ...Object.values(paths),
+    ...Object.values(circles),
+    {
```

Comment:
@bhavyajain511 I don't think this `{REF}` and `{VAL}` is needed, cause this is already there in the json which is getting imported

Reply:
Refer the `capacitor_horz` file